### DISCUSSION TITLE
No option to close with a comment?

### DISCUSSION THREAD

**Discussion by Zeko369** [community/community#86993] [Mobile]
It looks like I can't close an issue or PR with a comment in the app ☹️

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.144.0 (119864029)
Device: iPhone16,2 (iOS 17.3)
</details>

**Comment by mattruggio** [comment]
👋 @Zeko369. You are correct, currently closing an issue or PR and adding comments are split interactions on mobile while on dotcom there is an option to combine them. Related: our mobile applications are also backed by our GraphQL API for which there is no combined API for closing & commenting an issue/pr. I will add an internal issue to track this API & UI/UX parity mis-match between mobile and web.
